easybuild/easyconfigs/c/Clang/Clang-11.0.0-GCCcore-9.3.0.eb

```
@@ -55,11 +56,13 @@ builddependencies = [
     ('CMake', '3.16.4'),
     ('libxml2', '2.9.10'),
     ('Python', '3.8.2'),
+    ('SWIG', '4.0.1'),
```

perhaps also add a comment stating that it's a dep for lldb?
```suggestion
    ('SWIG', '4.0.1'),  # Optional dependency for LLDB
```